README.md

```diff
@@ -38,46 +68,77 @@ npm install react-native-in-app-notification --save
 | notificationBodyComponent | **See below about NotificationBody**                | React Node or Function | Recommended | `./DefaultNotificationBody` |
 
 ### NotificationBody
+
 The notification body is what is rendered inside the main Notification component and gives you the ability to customise how the notification looks. You can use the default notification body component in `./DefaultNotificationBody.js` as inspiration and guidance.
 
 Your `notificationBodyComponent` component is given four props:
 
 | Prop Name | Prop Description                                              | Data Type           | Default |
-|-----------|---------------------------------------------------------------|---------------------|---------|
+| --------- | ------------------------------------------------------------- | ------------------- | ------- |
 | title     | The title passed to `NotificationRef.show`                    | String              | `''`    |
 | message   | The message passed to `NotificationRef.show`                  | String              | `''`    |
 | onPress   | The callback passed to `NotificationRef.show`                 | Function            | `null`  |
 | icon      | Icon for notification passed to `NotificationRef.show`        | ImageSourcePropType | `null`  |
 | vibrate   | Vibrate on show notification passed to `NotificationRef.show` | Boolean             | `true`  |
 
 ## Usage
-Adding `react-native-in-app-notification` is simple; just import the component and add it to the bottom of your component tree. Then create a ref to the component using `ref={(ref) => { this.notification = ref; }}` as a prop.
 
-When you want to show the notification, just call `.show` on the ref you made earlier. `.show` can take three arguments: `title`, `message` and `onPress` all of which are optional - but you should probably include at least one of `title` or `message`! `onPress` doesn't need to be used for passive notifications and you can use `onClose` in your `NotificationBody` component to allow your users to close the notification.
+Adding `react-native-in-app-notification` is simple;
+Just wrap you main `App` component with the `Provider` component exported from this module.
+
+```javascript
+import { Provider } from 'react-native-in-app-notification';
```

Comment:
If people are using Redux could be annoying having to rename one of the `Provider` components, can you rename `Provider` here to something else like `NotificationProvider` or something similar

Reply:
Ok, do you think something like `InAppNotificationProvider` would work?
I tried to leave this choice to the module users, I used the common name so that they could import-rename it.